lib/active_merchant/billing/gateways/conekta.rb

```diff
@@ -197,8 +197,9 @@ def response_error(raw_response)
       end
 
       def json_error(raw_response)
-        msg = 'Invalid response received from the Conekta API.'
-        msg += "  (The raw response returned by the API was #{raw_response.inspect})"
+        #msg = 'Invalid response received from the Conekta API.'
+        #msg += "  (The raw response returned by the API was #{raw_response.inspect})"
```

Comment:
These can be deleted. Please don't submit PRs with commented out code, it just means we have to go in and remove it when we merge.